### PR TITLE
Add Aras Swag opportunity

### LINKED ISSUE #133: Add Aras to the list
- [x] I've checked that this isn't a duplicate issue.

Is it regarding the current promotions or something new?
Fill the corresponding section below:

Aras has started  with their campaign 
http://community.aras.com/en/join-aras-labs-hacktoberfest-2018/ 
You can add it to the list with proper difficulty and emoji's :100: 
Reference Link:
- [x] I've confirmed that the promotion is legit.
- [x] I've verified that it's still available to the public.

### PR DESCRIPTION
**Pull request by ncallan2019**
- [x] I've checked that this isn't a duplicate pull request.

* Company: Aras
* Reference URL: https://community.aras.com/en/join-aras-labs-hacktoberfest-2018/

Q. Have you received swag from this?  I have not.
Q. How long did it take?  N/A
Q. Does it ship worldwide?  N/A

Closes #133

### CODE REVIEW

File: data.json [hunk 1]
```diff
@@ -127,4 +127,12 @@
     "image": "https://developers.whatismybrowser.com/static/main/content/api/swag/whatismybrowser-stickers.jpg",
     "tags": ["stickers"]
   }
+  {
+    "name": "Aras",
+    "difficulty": "medium",
+    "description": "Submit an improvment to any <a href="http://community.aras.com/en/projects/">Aras Community Project</a> in October 2018 to earn stickers! Submit five, earn a limited edition T-shirt!",
```
Comment (vikaspotluri123) [inline data.json]: Double quotes need to be escaped within a string 😄😉

File: data.json [hunk 2]
```diff
@@ -127,4 +127,12 @@
...
+    "reference": "community.aras.com/en/join-aras-labs-hacktoberfest-2018/",
+    "image": "i-cdn.embed.ly/1/display/crop?height=300&key=fd92ebbc52fc43fb98f69e50e7893c13&url=https%3A%2F%2Fhacktoberfest.digitalocean.com%2Fassets%2Fhacktoberfest-2018-social-card-c8d2e1489f647f2e0a26e6f598adeb760872818905b34cd437afc7ac2857ceab.png&width=325",
```
Comment (vikaspotluri123) [inline data.json]: As with #131, we need to determine a better way to handle opportunities without explicit images - putting the hacktoberfest shirt instead will get repetitive really quickly 🤔

File: data.json [hunk 3]
```diff
@@ -127,4 +127,12 @@
     "image": "https://developers.whatismybrowser.com/static/main/content/api/swag/whatismybrowser-stickers.jpg",
     "tags": ["stickers"]
   }
+  {
+    "name": "Aras",
+    "difficulty": "medium",
+    "description": "Submit an improvment to any <a href="http://community.aras.com/en/projects/">Aras Community Project</a> in October 2018 to earn stickers! Submit five, earn a limited edition T-shirt!",
+    "reference": "community.aras.com/en/join-aras-labs-hacktoberfest-2018/",
```
Comment (aslafy-z) [inline data.json]: This link misses the protocol prefix: `http(s)://`

File: data.json (hunk 2 above)
Comment (aslafy-z) [inline data.json]: This link also

Comment (ncallan2019) [reply data.json]: When I try to use HTTPS I get an invalid certificate prompt... might just be Kaspersky.

Comment (aslafy-z) [reply data.json]: So use "http://" instead. It looks that their certificate points to the wrong domain.

File: data.json [hunk 4]
```diff
@@ -127,4 +127,12 @@
     "image": "https://developers.whatismybrowser.com/static/main/content/api/swag/whatismybrowser-stickers.jpg",
     "tags": ["stickers"]
   }
```
Comment (aslafy-z) [inline data.json]: You are missing a coma (`,`) in here.

File: data.json (hunk 1 above)
Comment (plibither8) [reply data.json]: Use single quotes instead!

File: data.json (hunk 4 above)
Comment (ncallan2019) [reply data.json]: I'm sorry, but I'm not sure where one is supposed to be.

Comment (aslafy-z) [reply data.json]:
```suggestion
  },
```

File: data.json [hunk 5]
```diff
@@ -130,8 +130,8 @@
   {
     "name": "Aras",
     "difficulty": "medium",
-    "description": "Submit an improvment to any <a href=\"community.aras.com/en/projects/\">Aras Community Project</a> in October 2018 to earn stickers! Submit five, earn a limited edition T-shirt!",
-    "reference": "community.aras.com/en/join-aras-labs-hacktoberfest-2018/",
+    "description": "Submit an improvment to any <a href=\"http://community.aras.com/en/projects/\">Aras Community Project</a> in October 2018 to earn stickers! Submit five, earn a limited edition T-shirt!",
+    "reference": "http://community.aras.com/en/join-aras-labs-hacktoberfest-2018/",
     "image": "https://www.aras.com/-/media/Images/Content-Row-Images/im-an-innovator.ashx?h=215&la=en&w=302&hash=CE2436EA9437F2B010DB4C1218CE48457AE30C43",
```
Comment (aslafy-z) [inline data.json]: Please use this image URL instead (no params & known extension):

https://www.aras.com/-/media/Images/Content-Row-Images/im-an-innovator.png

File: data.json (hunk 1 above)
Comment (aslafy-z) [reply data.json]: @ncallan2019 Please use @plibither8's solution instead

**Comment by aslafy-z** [comment]
Great, two more steps and we will be good:
- Use single quotes: https://github.com/swapagarwal/swag-for-dev/pull/135#discussion_r226825792
- Rebase to master

**Comment by ncallan2019** [comment]
Just wanted to thank everyone for helping a first-timer figure out his JSON.

**Comment by swapagarwal** [comment]
Seems like we're becoming a good fit to be added here: https://up-for-grabs.net/#/
See the bottom section:
```
I maintain a project!
We're looking for projects who can take the time out to help mentor developers as they get started with open source. 
```
Cheers to the community! 🙏

**Comment by aslafy-z** [comment]
Thanks @ncallan2019! :tada:

**Pull request merged**